### PR TITLE
chore(starters): add gatsby-starter-default-dark-mode

### PR DESCRIPTION
**Pull request by alexandreramosdev**

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -5161,3 +5161,15 @@
     - React Scroll for one page, anchor based navigation is available
     - Code highlighting
     - SEO optimized to include social media images and twitter handles
+- url: https://gatsby-starter-default-dark-mode.netlify.com/
+  repo: https://github.com/alexandreramosdev/gatsby-starter-default-dark-mode
+  description: A simple starter to get up and developing quickly with Gatsby, dark mode a styled-components.
```
Comment (lachlanjc) [inline docs/starters.yml]: Typo here :) Maybe this?
```suggestion
  description: A simple starter to get developing quickly with Gatsby, dark mode, and styled-components.
```

Comment (alexandreramosdev) [reply docs/starters.yml]: Thanks!  My english not is better yet

**Comment by laurieontech** [comment]
Thanks for the PR! I noticed that the README at the github link is still set up for the default starter. If you update that we'll be able to get this merged.

**Comment by alexandreramosdev** [comment]
> Thanks for the PR! I noticed that the README at the github link is still set up for the default starter. If you update that we'll be able to get this merged.

I forgot to switch the link. I already updated. 

Thanks

**Comment by laurieontech** [comment]
Awesome, thanks!

**Pull request merged**